docs/source/_json/types_document.json

```diff
@@ -0,0 +1,1298 @@
+GET /plone/@types/Document
+Accept: application/json
+
+HTTP 200 OK
+content-type: application/json+schema
+
+{
+  "properties": {
+    "allow_discussion": {
+      "choices": [
+        [
+          "True", 
+          "Yes"
+        ], 
+        [
+          "False", 
+          "No"
+        ]
+      ], 
+      "description": "Allow discussion for this content object.", 
+      "enum": [
+        "True", 
+        "False"
+      ], 
+      "enumNames": [
+        "Yes", 
+        "No"
+      ], 
+      "title": "Allow discussion", 
+      "type": "string"
+    }, 
+    "changeNote": {
+      "description": "Enter a comment that describes the changes you made.", 
+      "title": "Change Note", 
+      "type": "string"
+    }, 
+    "contributors": {
+      "additionalItems": true, 
+      "description": "The names of people that have contributed to this item. Each contributor should be on a separate line.", 
+      "items": {
+        "description": "", 
+        "title": "", 
+        "type": "string"
+      }, 
+      "title": "Contributors", 
+      "type": "array", 
+      "uniqueItems": true
+    }, 
+    "creators": {
+      "additionalItems": true, 
+      "description": "Persons responsible for creating the content of this item. Please enter a list of user names, one per line. The principal creator should come first.", 
+      "items": {
+        "description": "", 
+        "title": "", 
+        "type": "string"
+      }, 
+      "title": "Creators", 
+      "type": "array", 
+      "uniqueItems": true
+    }, 
+    "description": {
+      "description": "Used in item listings and search results.", 
+      "minLength": 0, 
+      "title": "Summary", 
+      "type": "string"
+    }, 
+    "effective": {
+      "description": "If this date is in the future, the content will not show up in listings and searches until this date.", 
+      "title": "Publishing Date", 
+      "type": "string"
+    }, 
+    "exclude_from_nav": {
+      "default": false, 
+      "description": "If selected, this item will not appear in the navigation tree", 
+      "title": "Exclude from navigation", 
+      "type": "boolean"
+    }, 
+    "expires": {
+      "description": "When this date is reached, the content will nolonger be visible in listings and searches.", 
+      "title": "Expiration Date", 
+      "type": "string"
+    }, 
+    "language": {
+      "choices": [
```

Comment:
Maybe we can reduce the number of available languages in the test fixture to make the auto-generated document more readable.